### ISSUE TITLE
Shall we use this https://tokens.coingecko.com/uniswap/all.json to replace our own default token list 

### ISSUE THREAD

**Issue by zhangzhongnan928** [AlphaWallet/alpha-wallet-ios#2769]
https://tokenlists.org/token-list?url=https://tokens.coingecko.com/uniswap/all.json

**Comment by colourfreak** [comment]
@zhangzhongnan928 We have most of token icons in .svg format. If we go with them, we might end up with poor .png quality. 
@hboon what do you think?

**Comment by zhangzhongnan928** [comment]
> @zhangzhongnan928 We have most of token icons in .svg format. If we go with them, we might end up with poor .png quality.
> @hboon what do you think?

for the ones we already have, don't replace.

**Comment by hboon** [comment]
Yes. If we have a curated/hand crafted one, we should use it. I’ll look into the list itself. There’s also more things we can do with it. Will check
